CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Open Liberty
+Anyone can contribute to the Open Liberty project and we welcome your contributions!
+
+There are multiple ways to contribute: report bugs, fix bugs, contribute code, improve upon documentation, etc.  You must follow these guidelines:
+* [Raising issues](https://github.com/OpenLiberty/stack/blob/master/CONTRIBUTING.md#raising-issues)
+* [Contributor License Agreement](https://github.com/OpenLiberty/stack/blob/master/CONTRIBUTING.md#contributor-license-agreement)
+* [Coding Standards](https://github.com/OpenLiberty/stack/blob/master/CONTRIBUTING.md#coding-standards)
+
+## Raising issues
+Please raise any bug reports on the [Open Liberty project repository's GitHub issue tracker](https://github.com/OpenLiberty/stack/issues). Be sure to search the list to see if your issue has already been raised.
```

Comment:
this URL is wrong..probably in the maven one too.

Reply:
I opened a stack issue about cleaning up CONTRIBUTING.md and other things you listed as part of this review (https://github.com/OpenLiberty/application-stack/issues/129). Doing this because the base impl for this repo was a straight copy of the main stack repo. The issue will address, links, CLA process that seems to be updates to be specific for the stack, copyrights, and add test/stack-test-inner-loop.sh